src/modules/stackoverflow/stackoverflow.api.js

```diff
@@ -0,0 +1,22 @@
+class StackoverflowAPI {
+    constructor(textValue) {
+        this.url = 'https://api.stackexchange.com/2.3/search/advanced?site=stackoverflow&q='
+        // this.url=`https://api.stackexchange.com/2.3/similar?&title=${textValue}&site=stackoverflow` // nie wiem czemu ta opcja nie działa, znalazłam alternatywę (chociaż teraz szuka po jakichś tagach, nie tytule
```

Comment:
Zwróć uwagę, że jest inna zasada działania, w pierwszej opcji doklejasz dane, w drugiej musisz osadzić w środku.
Żeby to zrobić musisz mieć dane po wpisaniu przez użytkownika, a nie po utworzeniu obiektu.

Najlepiej osobna metoda np. 
```
getUrl(textValue) { 
    // usunąłem też & który jest niepotrzebny
    return `https://api.stackexchange.com/2.3/similar?title=${textValue}&site=stackoverflow`
}

load(title) {
        return fetch(getUrl(title))
```